programs/ziti-edge-tunnel/include/model/dtos.h

```diff
@@ -90,6 +91,11 @@ XX(Duration, int, none, Duration, __VA_ARGS__) \
 XX(StartTime, timestamp, none,StartTime, __VA_ARGS__) \
 XX(Identities, tunnel_identity, array, Identities, __VA_ARGS__)
 
+#define INSTANCE_STATUS(XX, ...) \
+XX(ok, __VA_ARGS__)
```

Comment:
there is only one value in this enum. should it also at least have a "not ok" value? and if so, could it be replaced with a `bool`?

Reply:
added additional field "initialized" in this enum